app/src/main/java/org/oppia/android/app/player/state/SelectionInteractionView.kt

```diff
@@ -65,6 +63,7 @@ class SelectionInteractionView @JvmOverloads constructor(
     //  this more efficiently and cleanly than always relying on notifying of potential changes in the adapter when the
     //  type is set (plus the type ought to be permanent).
     this.selectionItemInputType = selectionItemInputType
+    singleTypeBuilderFactory = BindableAdapter.SingleTypeBuilder.Factory(Fragment())
```

Comment:
Ditto.

Reply:
Has been resolved in latest changes.

Reply:
@BenHenning yeah it is resolved i would say, it now works as expected with the changes. Thanks.